src/stores/game.js

```diff
@@ -515,7 +515,7 @@ export const useGameStore = defineStore('game', {
       await this.makeSocketRequest('draw');
     },
     async requestPlayPoints(cardId) {
-      await this.makeSocketRequest('points', { cardId });
+      await this.makeSocketRequest('points', { cardId, moveType: 'points', playedBy: this.myPNum });
```

Comment:
playedBy will be computed based on the requesting user's session and won't be included in the req body
```suggestion
      await this.makeSocketRequest('points', { cardId, moveType: 'points' });
```

Also nitpick but I like the convention of putting the moveType first since that will always be present and dictates what else goes in